src/components/Login.tsx

```diff
@@ -0,0 +1,107 @@
+import { useForm } from 'react-hook-form';
+import { useEffect, useState } from 'react';
+import { ViewOffIcon, ViewIcon } from '@chakra-ui/icons';
+import passwordValidation from '../utils/passwordValidation';
+import AuthButton from './common/Auth/AuthButton';
+import {
+  Input,
+  Center,
+  Text,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  InputRightElement,
+  InputGroup,
+} from '@chakra-ui/react';
+
+interface LoginFormValues {
+  email: string;
+  password: string;
+}
+
+const Login = () => {
+  const [isShow, setIsShow] = useState(false);
+  const {
+    register,
+    handleSubmit: onSubmit,
```

Comment:
```suggestion
    handleSubmit,
```

`handleSubmit`을 `onSubmit`으로 바꾸는 것도 좋지만 기존에 있는 함수를 굳이 바꿔서 사용할 필요는 없을 것 같습니다!!
이 `Login` 컴포넌트에 바꾸신 `onSubmit`과 `handleSubmit`을 자연스러운 흐름대로 다시 바꾸시는것이 어떨까요??

Reply:
수정완료했습니다👍